lib/bundix.rb

```diff
@@ -110,6 +107,10 @@ def self.object2nix(obj, level = 2, out = '')
     end
   end
 
+  def sh(*args)
+    self.class.sh(*args)
+  end
```

Comment:
You could also define the main method here and then use `module_function(:sh)` to export it to the module level.